getting-started/installation-apt-debian.md

```diff
@@ -37,22 +37,18 @@ sudo apt-get update
 sudo apt-get install timescaledb-postgresql-:pg_version:
 ```
 
-#### Update `postgresql.conf`
+#### Configure your database
 
->:TIP: The usual location of `postgres.conf`
-is `/etc/postgresql/:pg_version:/main/postgresql.conf`, but this may vary
-depending on your setup. If you are unsure where your `postgresql.conf` file
-is located, you can query PostgreSQL with any database client (e.g., `psql`)
-using `SHOW config_file;`.
-
-You will need to edit your `postgresql.conf` file to include
-necessary libraries:
+There are a [variety of settings that can be configured][config] for your
+new database. The easiest way to get started is to run `timescaledb-tune`,
```

Comment:
Only comment is that in this paragraph, it's not clear that you MUST use timescale-tune to put in the shared_preload_library, so maybe a slight re-word. You will need to update your `postgresql.conf` file to include necessary libraries. The easiest way to get started is to run `timescale-tune`, which is installed by default when using `apt`

Reply:
Updated.